protopipe/pipeline/event_preparer.py

```diff
@@ -57,10 +57,11 @@
 
 from scipy.sparse.csgraph import connected_components
 
+
 def camera_radius(cam_id=None):
```

Comment:
Why don't you just pass in the `SubarrayDescription` (or `TelescopeDescription` that you get from the MC file here? They way you do it here is just loading the pre-generated camera files (which I think were generated for Prod3, but not sure), so it may break if you switch to another prod. 

For example:
```py3
from functools import lru_cache # so we don't recompute it each time

@lru_cache
def cam_radius_deg(telescope: TelescopeDescription) → float:
    # compute it from telescope.camera.pix_x, pix_y..., telescope.optics.equivalent_focal_length
    return radius

# use it as follows:
telescope = event.inst.tel[tel_id]
radius = camera_radius_deg(telescope)
```

Reply:
Another question is if you need it in degrees at all, since you apply a cut as a fraction of that radius, right? Then it's the same in meters and degrees.

Reply:
For the moment I keep it in degrees because those are the same reference values used in CTA-MARS, also in which there is a conversion in meters afterwards. The best would be of course to understand how to translate the calculation of such values using only ctapipe.
For the moment I rewrote them as a dictionary.

Reply:
I modified this part in order to get the equivalent focal lengths using _ctapipe_.

The process using `lru_cache` fails with the following error, so for the moment, I didn't put it.

```
Traceback (most recent call last):
  File "/Users/michele/Applications/ctasoft/protopipe/protopipe/scripts/write_dl1.py", line 13, in <module>
    from protopipe.pipeline import EventPreparer
  File "/Users/michele/Applications/ctasoft/protopipe/protopipe/pipeline/__init__.py", line 5, in <module>
    from .event_preparer import *
  File "/Users/michele/Applications/ctasoft/protopipe/protopipe/pipeline/event_preparer.py", line 64, in <module>
    def camera_radius(camid_to_efl, cam_id="all"):
  File "/Users/michele/anaconda3/envs/protopipe/lib/python3.7/functools.py", line 490, in lru_cache
    raise TypeError('Expected maxsize to be an integer or None')
TypeError: Expected maxsize to be an integer or None
```